tests/unit/task/test_diagnostic_task.py

```diff
@@ -39,20 +38,17 @@ def test_initialize_env(ext, tmp_path, monkeypatch):
     env = {}
     if ext in ('.jl', '.py'):
         env['MPLBACKEND'] = 'Agg'
+    if ext in ('.ncl', '.R'):
+        env['diag_scripts'] = str(diagnostics_path)
     if ext == '.jl':
-        julia_path = f"{esmvaltool_path / 'install' / 'Julia'}"
+        task.env = dict(task.env)
+        env['JULIA_LOAD_PATH'] = f"{esmvaltool_path / 'install' / 'Julia'}"
 
         # check for new type of JULIA_LOAD_PATH
-        # and prepend new path arguments @:@$CONDA_ENV:@stdlib
+        # and cut away new path arguments @:@$CONDA_ENV:@stdlib
         # see https://github.com/ESMValGroup/ESMValCore/issues/1443
-        julia_project = os.environ.get('JULIA_PROJECT')
-        if julia_project is not None:
-            extras = ':@:' + julia_project + ':@stdlib'
-            julia_path = julia_path + extras
-
-        env['JULIA_LOAD_PATH'] = julia_path
-    if ext in ('.ncl', '.R'):
-        env['diag_scripts'] = str(diagnostics_path)
+        task.env['JULIA_LOAD_PATH'] = \
+            task.env['JULIA_LOAD_PATH'].split(":")[0]
```

Comment:
I would try to avoid modifying task.env in-place.

Reply:
well you can't check for the truncated `JULIA_LOAD_PATH` otherwise, since the `task.env` dict contains the one with the crazy `@:@$CONDA_ENV:@stdlib`

Reply:
Sure you can. Here are three ways:
```
test_env = copy.deep_copy(task.env)
...
```
```
env = ":".join(["..."] + task.env[...].split(":")[1:]
```
```
assert task.env[...].split(":")[0] == "..."
assert task.env["MPLBACKEN"] == "Agg"
...
```
My personal favorite is number three.

Reply:
ah with copying, yeh I don't like copying stuff, but I guess it saves us from changing it in place...OK then, I'll do it this way :+1:

Reply:
what you reckon, Klaus? The PITA Monitor test is dying coz of the pymon crappe, I still need to go back to that PR and figure out why those ancestors failures when using the `--db` flag, all else looks fine tho